templates/workloads/services/rd-web/cf.yml

```diff
@@ -122,3 +122,5 @@ Resources:
 
 {{include "addons" . | indent 2}}
 
+{{include "publish" . | indent 2}}
```

Comment:
just to double check: does this feature work for app runner 🤔

Reply:
It should!